reactful-numbers/src/app/components/ExpressionsBox.js

```diff
@@ -1,8 +1,6 @@
 import React from "react";
 
-function ExpressionsBox() {
-  return (
-    <input className="expressions-box" placeholder="Input" dir="rtl"></input>
-  );
+function ExpressionsBox({ value }) {
+  return <input className="expressions-box" value={value}></input>;
```

Comment:
The design seemed wrong, I will move the input to the right side of the input field with css later on